splinterd/src/daemon.rs

```diff
@@ -382,6 +388,24 @@ impl SplinterDaemon {
             rest_api_builder = rest_api_builder.add_resources(circuit_resource.resources());
         }
 
+        #[cfg(feature = "biome")]
+        {
+            info!("Adding biome routes");
+            let connection_pool: ConnectionPool =
+                database::create_connection_pool(&self.db_url).unwrap();
+            let mut biome_rest_provider_builder: BiomeRestResourceManagerBuilder =
+                Default::default();
+            biome_rest_provider_builder =
+                biome_rest_provider_builder.with_user_store(connection_pool.clone());
+            #[cfg(feature = "biome-credentials")]
+            {
```

Comment:
This shouldn't need the block (it is a single statement).

Reply:
This is a limitation with Rust. Specifically, it throws an error saying attributes cannot be used on expressions.